src/assets/js/global.js

```diff
@@ -366,6 +372,12 @@ String.prototype.hashCode = function() {
 			return ret;
 		}
 	};
+	
+	Number.prototype.valueBetween = function(lfs, rfs) {

```

Comment:
that also works. i just intendedly keep this file with no ES6 syntax for now... 😅